src/com/sun/ts/tests/jaxrs/spec/resourceconstructor/build.xml

```diff
@@ -17,23 +17,68 @@
 -->
 
 <project name="jaxrs_spec_resourceconstructor" basedir="." default="usage" >
+    <import  file="../../common/common.xml" />
 
     <property name="app.name"  value="jaxrs_spec_resourceconstructor" />
 
     <property name="resource.classes"
               value="com/sun/ts/tests/jaxrs/spec/resourceconstructor/Resource.class,
-    	com/sun/ts/tests/jaxrs/spec/resourceconstructor/HeaderResource.class,
-    	com/sun/ts/tests/jaxrs/spec/resourceconstructor/CookieResource.class,
-    	com/sun/ts/tests/jaxrs/spec/resourceconstructor/MatrixResource.class,
-    	com/sun/ts/tests/jaxrs/spec/resourceconstructor/QueryResource.class,
-    	com/sun/ts/tests/jaxrs/spec/resourceconstructor/PathResource.class,
-        com/sun/ts/tests/jaxrs/common/provider/PrintingErrorHandler.class"
+                     com/sun/ts/tests/jaxrs/spec/resourceconstructor/HeaderResource.class,
+                     com/sun/ts/tests/jaxrs/spec/resourceconstructor/CookieResource.class,
+                     com/sun/ts/tests/jaxrs/spec/resourceconstructor/MatrixResource.class,
+                     com/sun/ts/tests/jaxrs/spec/resourceconstructor/QueryResource.class,
+                     com/sun/ts/tests/jaxrs/spec/resourceconstructor/PathResource.class,
+                     com/sun/ts/tests/jaxrs/common/provider/PrintingErrorHandler.class"
     />
     <property name="appconfig.class"
               value="com/sun/ts/tests/jaxrs/spec/resourceconstructor/TSAppConfig.class"
     />
 
-    <import  file="../../common/common.xml" />
+    <target name="package" if="resource.classes">
```

Comment:
What is the problem being solved here?

Reply:
The "package" target used in these files is identical to the one imported from the common.xml, but with an added zipfileset stanza to pull in the beans.xml file.  This target will override the one that was used previously from the import.